.changeset/lucky-schools-enter.md

```
@@ -0,0 +1,7 @@
+---
+"@siemens/ix": minor
+---
+
+Selected chips in `ix-select` with `mode="multiple"` are now laid out on a single row. When the selected chips no longer fit the available width, the chips that would overflow are collapsed into a non-removable "+N" overflow indicator chip. Clicking (or activating via keyboard) the indicator opens a dropdown listing the hidden items, each of which can be removed.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)